nmmo/systems/inventory.py

```diff
@@ -173,17 +174,19 @@ def remove(self, item, quantity=None):
       stack = self._item_stacks[signature]
 
       if quantity is None or stack.quantity.val == quantity:
-        self.items.remove(stack)
+        self._remove(stack)
         del self._item_stacks[signature]
         return
 
       assert 0 < quantity <= stack.quantity.val, \
         f'Invalid remove {quantity} x {item} ({stack.quantity.val} available)'
       stack.quantity.val -= quantity
-
       return
 
+    self._remove(item)
+
+  # pylint: disable=protected-access
```

Comment:
why do we have to disable protected access?

Reply:
Oh, we don't Thanks!